### ISSUE TITLE
Show Edit Pod - Critical accessibility violation - Delete button does not have discernible text

### ISSUE THREAD

**Issue by sianford** [Sage/carbon#4535]
### Current behaviour

The Show Edit Pod has an accessibility violation "Ensures buttons have discernible text" due to the delete button icon not having inner text visible to screen readers.

See https://carbon.sage.com/?path=/story/showeditpod--default-story as an example.

### Expected behaviour

There are no accessibility violations.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/showeditpod--default-story

### JIRA Ticket (Sage Only)

FE-4445

### Carbon Version

95.1.1

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
FE-4445

**Comment by Parsium** [comment]
Closing this as `ShowEditPod` is now been deprecated
